jiraApi/jiraApi.go

```diff
@@ -23,13 +23,36 @@ import (
 	"github.com/sotomskir/jira-cli/jiraApi/models"
 	"github.com/spf13/viper"
 	"gopkg.in/resty.v1"
+	"regexp"
+	"strconv"
 	"strings"
 	"sync"
 	"time"
 )
 
+var clock clockwork.Clock
+
+func ParseDateTimeForJira(d string, t string, c clockwork.Clock) string {
+	md, _ := regexp.MatchString("\\d{4}-(((0)[0-9])|((1)[0-2]))-([0-2][0-9]|(3)[0-1])", d)
+	if d == "" || !md {
+		d = clock.Now().Format("2006-01-02")
+	}
+	mt, _ := regexp.MatchString("([01]\\d|2[0-3]):?([0-5]\\d)", t)
+	if t == "" || !mt {
+		t = clock.Now().Format("15:04")
```

Comment:
t is worklog start time, so we should subtract time spent from this.

Reply:
@sotomskir Is it really good idea? Most ppl will be updating timesheet at the end of the day. Also there is no limit to amount of time (12h for jira tempo) so if user wants to store 12h today at 08:00AM and he will not provide time flag our code with time subtract we will store this worklog yesterday :) and it will be confusing.

Reply:
Good point. Let's leave it as is.